### PR TITLE
Account management POC

### PR DESCRIPTION
**Pull request by dani-vh**
What's missing:
1. Remove email verification. ✅  
2. Differentiate active sesion by removing log-in/sign-up buttons and setting user name. ✅
3. Fix login -> log-in. ✅
4. Implement password resetting. ❌

### CODE REVIEW

**Comment by dani-vh** [comment]
@odarbelaeze let's leave the password resetting for another task, as well as providing ability to log-in through Google account.

**Comment by dani-vh** [comment]
BTW, @odarbelaeze it seems like the github action is stuck in an old commit, since it have been failing on the same thing even though I already fixed it, could you take a look? (I'm about to sleep :v), this is how it looks like locally (just fine):
image

File: src/components/layout/Header.tsx [hunk 1]
```diff
@@ -1,16 +1,45 @@
...
+          <TreeOfScience className="Header__branding" />
+          <h2 className="Header__title">Tree of Science</h2>
+        </div>
+      </Link>
+      <div className="Header_element">
+        {user?.uid ? (
+          <>
+            <p>{user.email}</p>
+            <div onClick={() => signOut(firebase.auth)}>
+              <h3 className="Header__log-out">Log Out</h3>
```
Comment (odarbelaeze) [inline src/components/layout/Header.tsx:27]: Let's use a buggon here

**Review by odarbelaeze — APPROVED** [review]
:shipit:

File: src/components/account/LogIn/hooks/useLogIn/index.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  });
+
+  const logIn = useCallback((data: LogInFormFieldsType) => {
+    signInWithEmailAndPassword(firebase.auth, data.email, data.password)
+      .then(() => {
+        setState({ status: "success", message: "Successfully logged in" });
+        navigate("/");
+      })
+      .catch((error) => {
+        console.log(JSON.stringify(error));
```
Comment (dani-vh) [inline src/components/account/LogIn/hooks/useLogIn/index.ts:32]: Lol, we forgot about this, life's hard @odarbelaeze

**Pull request merged**